subworkflows/local/input_check.nf

```diff
@@ -20,7 +22,6 @@ workflow INPUT_CHECK {
             meta -> tuple(meta.id[0], meta[0])
```

Comment:
I wonder if we should use the `meta.external_id` for checking for the when there are reads that need to be combined. As we created the `sample_name` column to allow for repeat values. `meta.id`  is used to here to find reads to be merged.
```
    grouped_tuples = reads_in.groupTuple(by: 0).branch {
            it ->
                merge_data: it[1].size() > 1
                format: true
            }
```

Reply:
This is a very good question, as the only way (after reverting `sample_name` to `sample`) to merge reads now would be if the IRIDANext ID is the same. But going forward reads are only getting merged within IRIDANext right? @apetkau

Reply:
After talking with Aaron, we decided to not let mikrokonod merge reads by default. We added a parameter to allow you too, but this will be a CLI feature. As in IRIDANext it is better to merge reads in the system where it is an auditable event and not something that may occur accidentally in a pipeline. 
but it is fixed here: https://github.com/phac-nml/mikrokondo/pull/140/commits/db5f420e82e289cd596817f48772b2e3323438a4